src/components/main-view.tsx

```diff
@@ -48,30 +50,50 @@ export const MainView: React.FC<IProps> = ({ showForce, mode, forceText, saveToT
           <div className="object-label" data-testid="object-1">Object 1: <SelectObject mode={mode} value={object1} onChange={handleSetObject1}/></div>
           <ObjectSymbol objectType={object1} />
         </div>
-        <div className="column arrows">
-          <div className="label">{forceText}</div>
-          { mode !== "vanilla" && !showForce
-              ? <div className="question-mark" data-testid="question-mark"
-                     title="Click on Calculate forces button to see a representation of the forces between the two objects"
-                     onClick={showAlert}>
-                     ?
-                </div>
-              : (mode === "earth")
-                ? <ForceValue object1={object1} object2={object2} />
-                : <ForceArrows object1={object1} object2={object2} />
-          }
+        <div className="column">
+          <div className="arrows">
+            { (mode !== "venus" || !showForce) && <div className="label">{forceText}</div> }
+            { mode !== "vanilla" && !showForce
+                ? <div className="question-mark" data-testid="question-mark"
+                      title="Click on Calculate forces button to see a representation of the forces between the two objects"
+                      onClick={showAlert}>
+                      ?
+                  </div>
+                : mode === "venus"
+                  ? <>
+                    <div className="label">Force between object 1 and Venus:</div>
+                    <ForceValue object1={object1} object2={object2} />
+                    <div className="label">Force between object 1 and Earth:</div>
+                    <ForceValue object1={object1} object2={object3} />
+                    </>
+                  : mode === "earth"
+                    ? <ForceValue object1={object1} object2={object2} />
+                    : <ForceArrows object1={object1} object2={object2} />
+            }
+          </div>
+          <div className="buttons">
+            <button onClick={handleSave} data-testid="save-button">{ mode !== "vanilla"
+                                                                     ? ( mode === "earth" ? "Calculate force" : "Calculate forces" )
+                                                                     : "Save to table"}
```

Comment:
Personally, I'm not a fan of nested `? :` syntax. It becomes a bit hard to follow, especially here where there's mix of `===` and `!==` etc. Ofc it's opinionated, but I guess simple sequence `if`, `switch`, or a hash with keys and values are usually easier to read when we have more combinations. Not asking to change it, just a random comment.